tests/gps/test_views.py

```diff
@@ -197,6 +197,11 @@ def test_beneficiary_details(client, snapshot):
                 "user_organization_id=[PK of organization]",
             ),
             ("href", f"beneficiary_id={beneficiary.pk}", "beneficiary_id=[PK of beneficiary]"),
+            ("hx-post", f"/gps/display/{group.pk}/{prescriber.public_id}/phone", "display_phone_url"),
+            ("hx-post", f"/gps/display/{group.pk}/{prescriber.public_id}/email", "display_email_url"),
+            ("id", f"phone-{prescriber.pk}", "display_phone_id"),
+            ("id", f"email-{prescriber.pk}", "display_email_id"),
```

Comment:
Pourquoi pas
```suggestion
            ("id", f"email-{prescriber.pk}", "email-[PK of prescriber]"),
```
et pareil pour les autres ?

Reply:
par pure flemme, mais c'est corrigé.